modules/library/main/src/main/java/org/geotools/data/ows/AbstractOpenWebService.java

```diff
@@ -448,7 +448,11 @@ protected Response internalIssueRequest(Request request) throws IOException, Ser
                 ByteArrayOutputStream out = new ByteArrayOutputStream();
                 request.performPostOutput(out);
                 try (InputStream in = getStream(out)) {
-                    httpResponse = httpClient.post(finalURL, in, postContentType);
+                	if(headers == null) {
```

Comment:
this is causing a QA failure because you used tabs, run `mvn spotless:apply` to fix

Reply:
Hi Mark,
yes, I actually fixed that yesterday already but it seems my second commit, pushed with "-force" did not update the PR. 
I'll have a look shortly. 
Thank you for jumping in!
Andreas

Reply:
Ok, fixed now...